consent/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+edX Enterprise's Consent application.
```

Comment:
In this, and perhaps also in some other docstrings for this app, can we please say more explicitly what "Consent" is? It's a very generic word and if I weren't part of the enterprise team I would have a hard time figuring out what this "consent" app is about.

Something like:
"The consent application provides an API that enables the platform to prompt the user to consent to share data about their activity and performance in (a) particular course(s) with the enterprise customer that is (usually) subsidizing their enrollment in those courses. When users grant consent to share data, that agreement is recorded in the database in an easily auditable way."